draft-ietf-ice-rfc5245bis.xml

```diff
@@ -2308,25 +2302,23 @@ typically have different component ID values.
 </section>
 
 <section title="Updating the Nominated Flag">
-
-<t>If the request had included a USE-CANDIDATE attribute in the
-Binding request, the controlling agent sets the Nominated Flag of the
-valid pair to true.  This concludes the ICE processing for this media
-stream; see <xref target="sec-conclude"/>.
+<t>If the controlling agent sends a Binding request with the USE-CANDIDATE
+attribute set, and if the agent receives a successful response to the request, 
+the agent sets the Nominated Flag of the pair to true. If the request fails
+<xref target="sec-client-failure"/>, the agent MUST remove the candidate pair 
+from the valid list, set the candidate pair state to Failed and set the check list 
+state to Failed.
 </t>
-
-<t>If the response was the result of a triggered check that was sent
-in response to a request that itself had the USE-CANDIDATE attribute,
-the controlled agent may now set the Nominated Flag for the pair
-learned from the original request.  This case is described in <xref
-target="sec-up-fav"/>.
+<t>If the controlled agent receives a successful response to a Binding request
+sent by the agent, and that Binding request was triggered by a received Binding
```

Comment:
s/agent/controlling agent/

Reply:
No, it is the response that was sent by the controlled agent, when it received the nomination request from the controlling agent.